src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_SequencePoints.vb

```diff
@@ -0,0 +1,91 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+
+Namespace Microsoft.CodeAnalysis.VisualBasic
+    Partial Class LocalRewriter
+        Friend Function AddConditionSequencePoint(condition As BoundExpression, containingStatement As BoundStatement) As BoundExpression
+            Dim local As LocalSymbol = Nothing
+            Return AddConditionSequencePoint(condition, containingStatement, local, shareLocal:=False)
+        End Function
+
+        Friend Function AddConditionSequencePoint(condition As BoundExpression, containingStatement As BoundStatement, ByRef lazyConditionalBranchLocal As LocalSymbol) As BoundExpression
+            Return AddConditionSequencePoint(condition, containingStatement, lazyConditionalBranchLocal, shareLocal:=True)
+        End Function
+
+        Private Function AddConditionSequencePoint(condition As BoundExpression,
+                                                   containingStatement As BoundStatement,
+                                                   ByRef lazyConditionalBranchLocal As LocalSymbol,
```

Comment:
Ok, it looks like we are using the input value. Then consider explicitly marking it as ```<[In], Out>```, this is applicable to other similar comments.